### PR TITLE
Import more double header sheets

### PR DESCRIPTION
**Pull request by ascii-dev**
## Description
Imports sheets with double headers from the 2015 report file.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/report_details.py [hunk 1]
```diff
@@ -111,9 +111,9 @@ def add_transnational_to_regions(regions):
     'ws_04': {
         '2015': {
             'name': '4',
-            'title': 'Topics in the news by region 1995-2015',
-            'desc': 'Breakdown of news topics cluster areas by region by media type',
-            'reports': ['global', 'region', 'country'],
+            'title': 'Topics in the news by region',
+            'desc': 'Breakdown of major news topics by region by medium',
+            'reports': ['global', 'region'],
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'reports': ['global', 'region', 'country'],
```

File: reports/report_details.py [hunk 2]
```diff
@@ -128,8 +128,8 @@ def add_transnational_to_regions(regions):
         '2015': {
             'name': '5',
             'historical': '5',
-            'title': 'Overall presence of women in news',
-            'desc': 'Summary of news subjects, by sex, by GMMP year',
+            'title': 'Summary of women in the news, by GMMP year',
+            'desc': 'Overall presence of women in news',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'title': 'Overall presence of women in news',
            'desc': 'Summary of news subjects, by sex, by GMMP year',
```

File: reports/report_details.py [hunk 3]
```diff
@@ -141,6 +141,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_06': {
+        '2015': {
+            'name': '6',
+            'title': 'Women in the news (sources) in major news topics by region',
+            'desc': '',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'desc': 'Breakdown of women in the news in major topics by region',
```

File: reports/report_details.py [hunk 4]
```diff
@@ -141,6 +141,13 @@ def add_transnational_to_regions(regions):
...
+            'reports': ['global', 'region'],
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'reports': ['global', 'region', 'country'],
```

File: reports/report_details.py [hunk 5]
```diff
@@ -466,6 +473,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_28': {
+        '2015': {
+            'name': '28',
+            'title': 'Female reporters, announcers and presenters all media, by region',
+            'desc': 'B.down of female reporters & presenters by region',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'desc': 'Breakdown of reporters & presenters by region',
```

File: reports/report_details.py [hunk 6]
```diff
@@ -466,6 +473,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_28': {
+        '2015': {
+            'name': '28',
+            'title': 'Female reporters, announcers and presenters all media, by region',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'title': 'Reporters, announcers and presenters all media, by region',
```

File: reports/report_details.py [hunk 7]
```diff
@@ -507,6 +535,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_32': {
+        '2015': {
+            'name': '32',
+            'title': 'Topics in the news - Detail by medium (ADD INTERNET) for female reporters',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'title': 'Topics in the news - Detail  by medium, by sex of reporter',
```

File: reports/report_details.py [hunk 8]
```diff
@@ -507,6 +535,13 @@ def add_transnational_to_regions(regions):
...
+            'desc': 'Breakdown of news topics by medium by sex of reporter',
+            'reports': ['global'],
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'reports': ['global', 'region', 'country'],
```

File: reports/report_details.py [hunk 9]
```diff
@@ -950,6 +1041,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_68b': {
+        '2015': {
+            'name': '68b',
+            'title': 'Twitter - Challenging Stereotypes',
+            'desc': '',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'desc': 'Twitter - This tweet clearly challenges gender stereotypes, by major topic',
```

**Review by kilemensi — APPROVED** [review]
:rocket: @ascii-dev 


-----------------------

In the queries gSheet, query `32` has the following comment:

>"- Include internet reporters(rather than tm_journalist_models, maybe journalist_models excluding twitter?)

Have we made the change/are we tracking it somewhere?

PS: `historical.json` is starting to become to **_heavy_**

**Comment by ascii-dev** [comment]
> 🚀 @ascii-dev
> 
> In the queries gSheet, query `32` has the following comment:
> 
> > "- Include internet reporters(rather than tm_journalist_models, maybe journalist_models excluding twitter?)
> 
> Have we made the change/are we tracking it somewhere?
> 
> PS: `historical.json` is starting to become to **_heavy_**

Hi @kilemensi, I tried looking in the queries gSheet but can't find where the comment is. Trying to understand what it means as I'm not sure I've made the change yet. I can include it in the last PR coming up.

**Comment by kilemensi** [comment]
> Hi @kilemensi, I tried looking in the queries gSheet ....

I've pinged you on the gSheet @ascii-dev

**Pull request merged**